src/STM32RTC.h

```diff
@@ -258,3 +261,959 @@ class STM32RTC {
 };
 
 #endif // __STM32_RTC_H
+
+
+#else /* C IMPLENTATION */
+
+/**
+ ******************************************************************************
+ * @file    STM32RTC.h
+ * @author  Zlobin Alexey
+ * @brief   Provides a RTC interface for Arduino
+ *
+ ******************************************************************************
+ */
+
+#ifndef STM32_RTC_H_
+#define STM32_RTC_H_
+
+#include "Arduino.h"
+#if defined(STM32_CORE_VERSION) && (STM32_CORE_VERSION  > 0x01090000)
+	#include "rtc.h"
+	#include "time.h"
+#endif
+// Check if RTC HAL enable in variants/board_name/stm32yzxx_hal_conf.h
+#ifndef HAL_RTC_MODULE_ENABLED
+	#error "RTC configuration is missing. Check flag HAL_RTC_MODULE_ENABLED in variants/board_name/stm32yzxx_hal_conf.h"
+#endif
+
+typedef unsigned char  u8;
+typedef unsigned short u16;
+typedef unsigned int   u32;
+typedef long unsigned int ul32;
+typedef unsigned long  u64;
+typedef signed char    i8;
+typedef signed short   i16;
+typedef signed int     i32;
+typedef signed long    i64;
```

Comment:
Avoid to define new type. Use standard one: uint8_t, int16_t,....
Please replace them in all definition and declaration